relayer/src/error.rs

```diff
@@ -82,6 +82,9 @@ pub enum Kind {
     #[error("Failed to create client {0}")]
     CreateClient(String),
 
+    #[error("Connection not found: {0}")]
+    ConnNotFound(ConnectionId),
```

Comment:
```suggestion
    ConnectionNotFound(ConnectionId),
```

Reply:
The thing is the remaining errors below are using the `Conn` short hand, `ConnOpen`, `ConnOpenInit`, `ConnOpenTry`, etc. Should I update them as well to make it consistent?

Reply:
No, because these errors are consistent with the names of their corresponding IBC message. Whereas `ConnNotFound` doesn't have a corresponding IBC message, and should instead be fully spelled out. Does that make sense?

Reply:
I still feel that it is better to be consistent with naming instead of following what is being called externally. But anyway I have renamed it as per suggestion.